website/src/pages/[organism]/submission/[groupId]/released.astro

```diff
@@ -39,10 +40,10 @@ const referenceGenomeSequenceNames = getReferenceGenomesSequenceNames(cleanedOrg
 
 const hiddenFieldValues = {
     [VERSION_STATUS_FIELD]: versionStatuses.latestVersion,
-    [GROUP_ID_FIELD]: group.groupId,
+    [GROUP_ID_FIELD]: String(group.groupId),
 };
 
-const initialQueryDict = Object.fromEntries(Astro.url.searchParams.entries());
+const initialQueryDict = parseUrlSearchParams(Astro.url.searchParams);
```

Comment:
@theosanderson Is this change here correct? I assume that you forgot to change it in #4914. Or did you leave it intentionally?

Reply:
I did not leave it intentionally – thanks!